.readme/partials/main.md.j2

```diff
@@ -16,7 +16,7 @@ You can follow this [tutorial](https://docs.stackable.tech/kafka/stable/getting_
 
 ## What Does This Do?
 
-This operator is managing Kafka in Kubernetes. Operators observe custom resources in a Kubernetes cluster, and manages the application it's responsible for according to the desired state specified in those custom resources.
+This operator is managing Kafka in Kubernetes. Operators observe custom resources in a Kubernetes cluster, and manage the application it's responsible for according to the desired state specified in those custom resources.
```

Comment:
This still reads wrong....

Maybe like this?
```suggestion
This operator is managing Kafka in Kubernetes. An operator observes custom resources in a Kubernetes cluster, and manages the application it's responsible for according to the desired state specified in those custom resources.
```

Reply:
This is just a suggestion but my earlier proposal still lead to bad english.

Reply:
Both read about right to me. Let's go with the proposed change.